### DISCUSSION TITLE
Getting UNABLE_TO_LOCK_ROW error when 2 records are processed at the same time.

### DISCUSSION THREAD

**Discussion by Ramyasree1421** [SFDO-Community/declarative-lookup-rollup-summaries#1433] [General]
Something went wrong for Order Insert failed. First exception on row 0; first error: CANNOT_INSERT_UPDATE_ACTIVATE_ENTITY, dlrs_RequestTrigger: execution of AfterInsert

caused by: System.DmlException: Update failed. First exception on row 0 with id  first error: UNABLE_TO_LOCK_ROW, unable to obtain exclusive access to this record or 1 records: 

(dlrs)
Trigger.dlrs_RequestTrigger: line 7, column 1

**Comment by aheber** [comment]
@Ramyasree1421 I see you also posted this in the Trailblazer Community which is great! Looks like you were able to track down that this was related to Certina's app changes.

It is unfortunate that DLRS is the entry point for various other actions so it appears to be the source of troubles when it is something deeper down.
